src/ResourceManager.cpp

```diff
@@ -0,0 +1,34 @@
+#include "ResourceManager.h"
+
+ResourceManager::ResourceManager() {
+	LoadData();
```

Comment:
Let's not call it in the constructor. Instead, make LoadData (or rather is should be LoadResources) public, and call it at the beginning of main function.
Otherwise the first call to Get will also cause all assets to be loaded - it won't be a problem in our game, but in an actual game it could cause a freeze for a second or two. So it's better to load everything before even the game start :)